contracts/core/CreatorCore.sol

```diff
@@ -269,42 +273,60 @@ abstract contract CreatorCore is ReentrancyGuard, ICreatorCore, ERC165 {
     /**
      * Helper to get royalties for a token
      */
-    function _getRoyalties(uint256 tokenId) view internal returns (address payable[] storage, uint256[] storage) {
-        return (_getRoyaltyReceivers(tokenId), _getRoyaltyBPS(tokenId));
+    function _getRoyalties(uint256 tokenId) view internal returns (address payable[] memory receivers, uint256[] memory bps) {
+
+        // Get token level royalties
+        RoyaltyConfig[] memory royalties = _tokenRoyalty[tokenId];
+        if (royalties.length == 0) {
+            // Get extension specific royalties
+            address extension = _tokensExtension[tokenId];
+            if (extension != address(0)) {
+                if (ERC165Checker.supportsInterface(extension, type(ICreatorExtensionRoyalties).interfaceId)) {
+                    (receivers, bps) = ICreatorExtensionRoyalties(extension).getRoyalties(address(this), tokenId);
```

Comment:
value in sanity check here? require tbps < 10000 && receivers.length === bps.length

Reply:
Don't know if I actually care, it's a read function.  This should be caught by anyone reading this data.